### DISCUSSION TITLE
Build agent into container image

### DISCUSSION THREAD

**Discussion by jrassi06** [actions/virtual-environments#7259] [Image configuration]
can the instructions be updated to reference how to create a container image for the runner agents (GitHub and ado)? I know VM is listed, but it would be helpful if container instructions were also included. thanks.

**Comment by erik-bershel** [comment]
I believe that feature would be unavailable for Hosted Agents due to global difference between conceptions of containers and VMs and that fact that images contain services like Docker inside.
